modules/olmv1-finding-operators-to-install.adoc

```diff
@@ -111,10 +112,62 @@ $ jq -s '.[] | select( .schema == "olm.package") | \
   "defaultChannel": "stable",
   "icon": {
     "base64data": "PHN2ZyB4bWxu..."
-    "mediatype": "image/svg+xml"
+    "mediatype": "image/png"
   },
-  "name": "serverless-operator",
+  "name": "openshift-pipelines-operator-rh",
   "schema": "olm.package"
 }
 ----
 ====
+
+. Currently, OLM 1.0 supports extensions that do not use webhooks and are configured to use the `AllNamespaces` install mode. Extensions that use webhooks or that target a single or specified set of namespaces cannot be installed. To search catalog for bundles that support `AllNamespaces` mode, run the following command:
++
+[source,terminal]
+----
+$ opm render --migrate <catalog_reference> | \
+  jq 'select(.schema == "olm.bundle" and (.properties[] | \
+  select(.type == "olm.csv.metadata" \
+  and (.value.installModes[] | \
+  select(.type == "AllNamespaces" and .supported == true) \
+  != null)) != null)).name'
+----
```

Comment:
@joelanford Can we modify this query to also search for bundles that do not include webhooks?